packages/nimble-components/src/table-column/anchor/cell-view/index.ts

```diff
@@ -26,11 +26,25 @@ TableColumnAnchorColumnConfig
     @observable
     public hasOverflow = false;
 
+    /** @internal */
+    @observable
+    public isPlaceholder = false;
+
     /** @internal */
     public anchor?: Anchor;
 
     @volatile
     public get text(): string {
+        const displayedText = this.cellRecord?.label ?? this.cellRecord?.href;
+        if (
+            (displayedText === undefined || displayedText === null)
+            && this.columnConfig?.placeholder
```

Comment:
Actually maybe empty string should be allowed as thee placeholder string?

Reply:
Sure, I can allow empty string as the placeholder and add some tests for it. I don't think it'll actually make a big difference from a client point of view (at least with our current columns) because values that would use the placeholder (`null`/`undefined`) get rendered as empty strings when the placeholder isn't configured. Therefore, an `undefined` placeholder and an empty placeholder would render the exact same way in a cell.